test/end-to-end/specs/test_createBlueprint.js

```diff
@@ -0,0 +1,110 @@
+const assert = require('assert');
+const faker = require('faker');
+const helper = require('../utils/helper');
+const testData = require('../wdio.conf.js').testData;
+
+const BlueprintsPage = require('../pages/blueprints');
+const DeleteBlueprintPage = require('../pages/deleteBlueprint');
+const CreateBlueprintPage = require('../pages/createBlueprint');
+const EditBlueprintPage = require('../pages/editBlueprint');
+const ChangesPendingCommitPage = require('../pages/changesPendingCommit');
+const ToastNotifPage = require('../pages/toastNotif');
+
+
+describe('Create Blueprint Page', () => {
+  const blueprintsPage = new BlueprintsPage();
+  const createBlueprintPage = new CreateBlueprintPage(
+    testData.blueprint.simple.name,
+    testData.blueprint.simple.description);
+
+  beforeEach(() => {
+    helper.goto(blueprintsPage);
+  });
+
+  describe('Input Data Validation Test', () => {
+    describe('Required Field Missing', () => {
+      it('show alert message when creating blueprint without name', () => {
+        browser
+          .click(blueprintsPage.btnCreateBlueprint)
+          .waitForVisible(createBlueprintPage.dialogRootElement);
+
+        // wait until the name input element has focus
+        browser
+          .waitUntil(() => $(createBlueprintPage.inputName).hasFocus());
+
+        browser
+          .waitForVisible(createBlueprintPage.btnCreate);
+
+        browser
+          .click(createBlueprintPage.btnCreate)
+          .waitForVisible(createBlueprintPage.labelAlertInfo);
+
+        const actualAlertText = $(createBlueprintPage.labelAlertInfo).getText();
+        const actualHelpText = $(createBlueprintPage.spanHelpBlockMsg).getText();
+
+        assert.equal(actualAlertText, createBlueprintPage.varAlertInfo);
+        assert.equal(actualHelpText, createBlueprintPage.varHelpBlockMsg);
+      });
+
+      it.skip('show alert message when creating blueprint without name by clicking Enter', () => {
```

Comment:
Should remove skip here if there's no special reason.

Reply:
there is, see the comment down below in the test. From what I can tell this is not supported on Firefox.